blog/_posts/2018-07-09-dvteam-activated.md

```diff
@@ -0,0 +1,82 @@
+---
+title: DV Team Activated!
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/2018-07-09-dvteam-activated
+current: blog
+author: Violeta
+twitter: artnerdnet
+categories: student-posts
+- news
+---
+
+## Starting a new adventure with RgSoC and Living Style Guide from coding newbies to professional front-end developers in 11 months.
+
+![image](https://pbs.twimg.com/media/DiX7VAYX4AEMD2v.jpg)
+
+### Better Together
+November 2017, we started as complete coding newbies, with as little coding knowledge as HTML can be. JavaScript, the word alone, felt like a entire new universe. We can’t even believe how much we have learned in the past 8 months! Coming from a non technical background and being able to code in so little time has been quite a journey. Yet, it made us realize we have definitely unknown abilities and a real passion for programming.
+
+We started with a selected group of 20.000 out of 200.000 applications in Europe, Africa and Middle East region. Earning a full Google Scholarship for Front End Web Developer at Udacity was the fruit of this three months of dedication, hard work, discipline, cooperation with peers coders and taking actively part in a huge social community, in order to stand out and win our spot in the top 2.000 students (out of 20.000). We are now about to finish the last part: React.
+
+### RgSoC Coaches & Supervisor
+During the Google Scholarship, we met several times in real life and decided to apply for RgSoc to be able to keep practicing our coding skills in summer. As we were new in the dev community, we reached out to our Udacity community managers who introduced us to Amanda Cavallaro. She helped us find our first two current RgSoc-coaches: 
+
+*Coach 1: Tobias (React developer)* 

```

Comment:
It looks bit weird to have bio for all other coaches except this one. :) Can you please add few lines here as well if you wish to keep mentioning this coach in the blog post?